src/higley_lab_to_nwb/benisty_2022/benisty_2022_spike2events_interface.py

```diff
@@ -75,34 +87,46 @@ def get_metadata(self) -> dict:
 
         return metadata
 
-    def add_to_nwbfile(self, nwbfile: NWBFile, metadata: dict) -> None:
+    def get_event_times_from_ttl(self, stream_id):
+        extractor = CedRecordingExtractor(file_path=str(self.source_data["file_path"]), stream_id=stream_id)
+        times = extractor.get_times()
+        traces = extractor.get_traces()
+        event_times = get_rising_frames_from_ttl(traces)
+        return times[event_times]
 
+    def add_to_nwbfile(self, nwbfile: NWBFile, metadata: dict) -> None:
         events_metadata = metadata["Events"]
-        
         ttl_types_table = TtlTypesTable(**events_metadata["TTLTypesTable"])
-
         ttls_table = TtlsTable(**events_metadata["TTLsTable"], target_tables={"ttl_type": ttl_types_table})
 
-        ttl_type = 0
-        for stream_id, stream_name in self.stream_ids_to_names_map.items():
-            extractor =  CedRecordingExtractor(file_path=str(self.source_data["file_path"]), stream_id=stream_id)
-            times = extractor.get_times()
-            traces = extractor.get_traces()
-            event_times = get_rising_frames_from_ttl(traces)
-
+        for ttl_type, (stream_id, stream_name) in enumerate(self.ttl_stream_ids_to_names_map.items()):
+            timestamps = self.get_event_times_from_ttl(stream_id=stream_id)
             ttl_types_table.add_row(
                 event_name=stream_name,
                 event_type_description=f"The onset times of the {stream_name} event.",
                 pulse_value=1,
             )
-
-            if len(event_times):
-                for timestamp in times[event_times]:
+            if len(timestamps):
+                for timestamp in timestamps:
                     ttls_table.add_row(
                         ttl_type=ttl_type,
                         timestamp=timestamp,
                     )
-            ttl_type+=1
 
         nwbfile.add_acquisition(ttl_types_table)
-        nwbfile.add_acquisition(ttls_table)
\ No newline at end of file
+        nwbfile.add_acquisition(ttls_table)
+
+        for stream_id, stream_name in self.behavioral_stream_ids_to_names_map.items():
+            extractor = CedRecordingExtractor(file_path=str(self.source_data["file_path"]), stream_id=stream_id)
+            gain, offset = _get_stream_gain_offset(file_path=str(self.source_data["file_path"]), stream_id=stream_id)
+            behavioral_time_series = TimeSeries(
+                name=stream_name,
+                data=extractor.get_traces(),
+                rate=extractor.get_sampling_frequency(),
+                description=f"The {stream_name} measured over time.",
+                unit="volts",
```

Comment:
https://nwbinspector.readthedocs.io/en/dev/best_practices/time_series.html?highlight=ElectricalSeries#units-of-measurement
```suggestion
                unit="Volts",
```